releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,22 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 4.0.0-preview - January 4, 2024
+
+### Breaking Changes
+
+There are no breaking changes in this release.
+
+### Enhancements
+
+* [Variants](https://github.com/microsoft/FeatureManagement-Dotnet/tree/release/v4?tab=readme-ov-file#variants) - Allows for more complex control of features than a simple on/off flag.
+* [Telemetry](https://github.com/microsoft/FeatureManagement-Dotnet/tree/release/v4?tab=readme-ov-file#telemetry) - Allows for analysis and tracking of feature assignment. 
+* [Status](https://github.com/microsoft/FeatureManagement-Dotnet/tree/release/v4?tab=readme-ov-file#status) - A new optional field on a Feature that controls how a flag's enabled state is evaluated. Flags can set this field to "Disabled" which will cause the flag to always act disabled, regardless of filters or other schema. 
+
+### Additional
+
+* An example was added to demonstrate how to use the new Telemetry in an ASP.NET application. See https://github.com/microsoft/FeatureManagement-Dotnet/tree/release/v4?tab=readme-ov-file#telemetry
```

Comment:
```suggestion
* An example was added to demonstrate how to use the new Telemetry function in an ASP.NET application. See https://github.com/microsoft/FeatureManagement-Dotnet/tree/release/v4?tab=readme-ov-file#telemetry
```

Reply:
The link seems to be duplicated here. Can we provide a link to the example or just remove this link?

Reply:
Oh oops, that's the wrong link. Thank you.